assignments/lambda-classes.js

```diff
@@ -32,6 +32,16 @@ class Instructor extends Person {
 	grade(student, subject) {
 		console.log(`${student.name} receives a perfect score on ${subject}`)
 	}
+	score(studentGrade) {
+		var totalScore = studentGrade + ((Math.random() * 201) - 100);
```

Comment:
Since you won't be changing the value of this variable during an individual method call, you can use `const` here.